### ISSUE TITLE
Add a changelog

### ISSUE THREAD

**Issue by mj-will** [gwastro/BBHX-waveform-model#12]
It would be useful to have a changelog to log breaking changes, such as the TDI change.

**Comment by WuShichao** [comment]
Yeah, how to do that? Make a new release and change the log with it?

**Comment by WuShichao** [comment]
I also noticed that you changed the approx name: `BBHX_PhenomD` to `BBHX_IMRPhenomD.`

**Comment by WuShichao** [comment]
I will create a PR to change the name back, otherwise I need to change the name in PyCBC PE example and last week's LISA demo.

**Comment by mj-will** [comment]
> I also noticed that you changed the approx name: `BBHX_PhenomD` to `BBHX_IMRPhenomD.` This will break PyCBC PE example.

Yes, this wasn't intentional and I think the change was introduced in https://github.com/gwastro/BBHX-waveform-model/pull/2 and then I copied it over. I'll add a test for this to avoid this in future.

**Comment by mj-will** [comment]
I realised PyCBC doesn't have a changelog either, so it's debatable whether this package needs one. If you were to add one, I'd follow some sort of standard, e.g. https://keepachangelog.com/en/1.1.0/
